docs/griptape-tools/official-tools/griptape-cloud-knowledge-base-client.md

```diff
@@ -0,0 +1,25 @@
+## Overview
+
+The `GriptapeCloudKnowledgeBaseClient` is an advanced Tool to retrieve data from a RAG pipeline and vector stores hosted in [Griptape Cloud](https://cloud.griptape.ai). It performs searches across a centralized [Knowledge Base](https://cloud.griptape.ai/knowledge-bases) that can consist of various data sources such as Confluence, Google Docs, and web pages.
```

Comment:
> It performs searches across a centralized
Similar, maybe reword to emphasize that this is a feature of the service, not the tool.